README.md

```diff
@@ -9,54 +9,54 @@
-|   | [`Symbol.prototype.description`](https://github.com/tc39/proposal-Symbol-description)                       | Michael Ficarra                    | 1 |
-|   | change Number.parseInt/parseFloat to not coerce null/undefined/NaN (repo link TBD)                          | Brendan Eich                       | 1 |
-|   | [Array.prototype.flat{Map,ten}](https://github.com/tc39/proposal-flatMap)                                   | Brian Terlson, Michael Ficarra     | 1 |
-|   | [Throw expressions](https://github.com/rbuckton/proposal-throw-expressions)                                 | Rob Buckton                        | 1 |
+| 🚀 | Proposal                                                                                                    | Champion                           | Stage | Tests |
+|---|-------------------------------------------------------------------------------------------------------------|------------------------------------|-------|-------|
+|   | [`Function.prototype.toString` revision](https://github.com/tc39/Function-prototype-toString-revision)      | Michael Ficarra                    | 3 | [x] [link](tc39/test262#1163) |
+|   | [`global`](https://github.com/tc39/proposal-global)                                                         | Jordan Harband                     | 3 | [x] [link](tc39/test262#765) |
+|   | [Rest/Spread Properties](https://github.com/tc39/proposal-object-rest-spread)                               | Sebastian Markbage                 | 3 | [ ] [link](tc39/test262#865) |
+|   | [Asynchronous Iteration](https://github.com/tc39/proposal-async-iteration)                                  | Domenic Denicola                   | 3 | [ ] [link](tc39/test262#923) |
+|   | [`import()`](https://github.com/tc39/proposal-dynamic-import)                                               | Domenic Denicola                   | 3 | [ ] [link](tc39/test262#1164) |
+|   | [RegExp Lookbehind Assertions](https://github.com/tc39/proposal-regexp-lookbehind)                          | Daniel Ehrenberg                   | 3 | [x] [link](tc39/test262#999) |
+|   | [RegExp Unicode Property Escapes](https://github.com/tc39/proposal-regexp-unicode-property-escapes)         | Brian Terlson, Daniel Ehrenberg, Mathias Bynens | 3 | [x] [link](tc39/test262#1014) |
+|   | [RegExp named capture groups](https://github.com/tc39/proposal-regexp-named-groups)                         | Daniel Ehrenberg, Brian Terlson    | 3 | [x] [link](tc39/test262#998) |
+|   | [`s` (`dotAll`) flag for regular expressions](https://github.com/mathiasbynens/es-regexp-dotall-flag)       | Mathias Bynens, Brian Terlson      | 3 | [x] [link](tc39/test262#997) |
+|   | [Legacy RegExp features in JavaScript](https://github.com/tc39/proposal-regexp-legacy-features)             | Mark Miller, Claude Pache          | 3 | [ ] [link](tc39/test262#1165) |
```

Comment:
this URL format doesn't work; it needs to be the full HTML link. We should probably define these links at the bottom of the document with footnotes, so the rows here don't need to be super long.